### DISCUSSION TITLE
LogContext: Open context in split view

### DISCUSSION THREAD

**Discussion by svennergr** [grafana/grafana#56390] [Observability]
The context of a logline is currently displayed in a modal which is opened in the center of the screen. A button to open the queries used to gather the logs of the context in a split view feels like a good thing to do.

**Comment by stevendungan** [comment]
@svennergr adding notes from our call with RAMP: https://docs.google.com/document/d/1ItN_Jsb-8iZoNH71xzynMqRjWCSkq-LuAYuxE2CCqrs/edit?usp=sharing

Your thoughts on "why" (https://raintank-corp.slack.com/archives/C03PMJLVC67/p1666113121704309?thread_ts=1666104985.328129&cid=C03PMJLVC67) make sense and seem to align with the customer is after - they just really disliked the split view idea, and their reasoning for why seems like it could apply broadly

Are you and the team open to re-working this where the spirit is the same, but actually delivery may not be split-view? Could see if Niat would be able to help generate some ideas, and maybe the customer is open to giving feedback on some mock-ups.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
